botocore/exceptions.py

```diff
@@ -295,9 +295,11 @@ class ClientError(Exception):
         'operation: {error_message}')
 
     def __init__(self, error_response, operation_name):
+        code = error_response.get('Error', {}).get('Code', 'Unknown')
```

Comment:
Good point. Will change back to error_response['Error'].get('Code', 'Unknown') in next PR. That way, shoud the 'Error' be missing in future, it deserves a lousy error trace. :)